crates/rolldown/src/module_loader/module_task.rs

```diff
@@ -228,7 +228,9 @@ impl ModuleTask {
       asset_view,
     };
 
-    self.ctx.plugin_driver.module_parsed(Arc::new(module.to_module_info())).await?;
+    let module_info = Arc::new(module.to_module_info());
+    self.ctx.plugin_driver.set_module_info(&module.id, Arc::clone(&module_info));
```

Comment:
It seems you set it twice, https://github.com/rolldown/rolldown/pull/2625/files#diff-c9c7714142367c1f6a60995eb1e3ec2255fcab1aa62141a33de5268a814bdf57R335-R338

Reply:
Yeah.Here beacause the `importers/dynamic_importers` changed. I have a try to spreate it to `PluginContext#moduleImporters`, and merge it at `PluginContext#getModuleId` called, but it is also not improving benchmark. The finally i found the performance regression caused by here.